tests/e2e/Services/Functions/FunctionsCustomClientTest.php

```diff
@@ -963,4 +963,52 @@ public function testNonOverrideOfHeaders(): array
 
         return [];
     }
+
+    public function testGetFunctionTemplates()
+    {
+        $templates = $this->client->call(Client::METHOD_GET, '/functions/templates', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+        ], $this->getHeaders()));
+
+        $this->assertEquals(200, $templates['headers']['status-code']);
+        $this->assertGreaterThan(0, $templates['body']['total']);
+        $this->assertIsArray($templates['body']['templates']);
+        $this->assertArrayHasKey('runtimes', $templates['body']['templates'][0]);
```

Comment:
Lets add test for exact responses. From test, we can look into config file as well. Let's compare response, with config itself. That ensures response filters work well.